crates/bevy_reflect/src/enums/enum_trait.rs

```diff
@@ -295,12 +296,16 @@ impl<'a> Iterator for VariantFieldIter<'a> {
 
 impl<'a> ExactSizeIterator for VariantFieldIter<'a> {}
 
+/// A field in the current enum variant.
 pub enum VariantField<'a> {
+    /// The name and value of a field in a struct variant.
     Struct(&'a str, &'a dyn PartialReflect),
+    /// The value of a field in a tuple variant.
     Tuple(&'a dyn PartialReflect),
 }
 
 impl<'a> VariantField<'a> {
+    /// Returns the name of a struct variant field, or [`None`] for a tuple variant field.
```

Comment:
Also none for a unit, no?

Reply:
Unlike `VariantType`, `VariantField` does not have a `Unit` variant. (I debated adding a note to this effect when writing this comment, but decided against it!)